hydromt/workflows/grid.py

```diff
@@ -147,8 +150,9 @@ def grid_from_raster_reclass(
     mask_name: Optional[str] = "mask",
     rename: Optional[Dict] = dict(),
 ) -> xr.Dataset:
-    """
-    Prepares data variable(s) resampled to grid_like object by reclassifying the data in ``da`` based on ``reclass_table``.
+    """Prepare data variable(s) resampled to grid_like object.
```

Comment:
Same comment as in the setup methods. the First line is what is printed in short docs api do good the full explanation is here. So would be nice to undo